tests/test_MocLogin.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = test_settings['username']
```

Comment:
As I tried to explain to you, the cyverse use case is different than ours.  We want to be as close to the one database policy, therefore, we automatically create the account on atmosphere after obtaining the token from open stack.

The UserFactory will create a user on atmosphere which will test cyverse's login to atmosphere but this is not our use case.  We need to test the log in without having any user credentials (or passwords) stored on atmosphere.  This implies that we actually need to simulate keystone in order to this with Travis CI.